### DISCUSSION TITLE
Question about pool rewards

### DISCUSSION THREAD

**Discussion by bluehawana** [Chia-Network/chia-blockchain#11206] [Support]
### What happened?

Hi, I joined spce pool to mining chia. I found 2 blocks on Spacepool but did not get reward on Chia wallet. I have checked my wallet address it was correct and I have received 2 0.25 chia rewards with the same settings. Please help me to figure it out, thanks. 

### Version

1,2,11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by shaneo257** [comment]
when your farming part of a pool, you only receive the farmer rewards ie 0.25 plus fees, the remaining 1.75 goes to the pool to be shared between all the users

**Comment by emlowe** [comment]
Moving to support question

**Reply by bluehawana** [reply]
Why nobody answer my question?

**Comment by bluehawana** [comment]
Please, I need your help.

**Comment by bluehawana** [comment]
Could any admin or support pls help me with the payment issue I have?
